### DISCUSSION TITLE
[feature] <Sheet native /> for iOS and android

### DISCUSSION THREAD

**Discussion by natew** [tamagui/tamagui#1006] [General]
This is pretty straightforward and can have a small bounty on it, get Sheet native prop to work for iOS to start should be fine and we can go from there.

**Comment by mphill** [comment]
Not sure this is really possible without a new API to Tamagui.

A few questions/comments:
- Is the goal to use something like https://reactnative.dev/docs/actionsheetios under hood?
- There is no android Action Sheet in React Native (https://stackoverflow.com/questions/35052767/whats-the-equivalent-of-actionsheetios-in-android) which would likely result in a dependency
- What does this do on web, a <select />?
- If the goal is to have the `native` prop do this: https://github.com/expo/react-native-action-sheet I think it gets quite complicated trying to interpret the components into the strings needs to the action sheet options.

**Comment by natew** [comment]
We'd need basically this library:

- https://github.com/dominicstop/react-native-ios-modal

But to get it to support expo modules + snap points (though could ship without snappoints to start and just warn).

On the web it wouldn't do anything differently since there is no Sheet concept on the web.

**Comment by natew** [comment]
Keep in mind this isn't ActionSheet just Sheet. ActionSheet is something we want to add eventually.
